src/common/pico_sync/sem.c

```diff
@@ -6,6 +6,7 @@
 
 #include "pico/sem.h"
 #include "pico/time.h"
+#include "sys/select.h"
```

Comment:
???
This PR seems like a strange place for this change to be included? :wink:

Reply:
doh! i was looking at something else, and was wondering if it was available